packages/runtime/container-runtime/README.md

```diff
@@ -51,6 +51,52 @@ issue](https://github.com/microsoft/FluidFramework/wiki/Submitting-Bugs-and-Feat
 
 Thank you!
 
+## Data Virtualization For DataStores (Work in Progress)
+
+It's a capability to exclude some content from initial snapshot (used when loading container) and thus improve boot
+performance of a container. Excluded content could be loaded at a later time when it's required.
+
+### Motivation for Data Virtualization
+
+This section talks about how the system used to work before Data virtualization. Fluid (through its APIs) does expose
+some levels of data virtualization. But under covers, there is almost none. The whole content of the file is downloaded
+in one go. Due to limitation of data virtualization, FF holds ALL blobs in snapshot as those might be required in the
+future. Any delayed loading (through FF APIs) results in loading state of DDSs at a sequence number of snapshot we
+booted from, and application of all the ops up till current sequence number. While application may choose not to load
```

Comment:
>and application of all the ops up till current sequence number. [](http://example.com/codeflow?start=13&length=63)

this sentence doesn't read/fit well with what precedes it.

Reply:
Didn't get it? What part does not fit well?

Reply:
discussed on chat.